runtime.md

```diff
@@ -9,8 +9,11 @@ $ funC [global-options] <COMMAND> [command-specific-options] <command-specific-a
 
 ## Global options
 
-None are required, but the runtime may support options that start with at least one hyphen.
-Global options may take positional arguments (e.g. `--log-level debug`), but the option parsing must be such that `funC <COMMAND>` is unambiguously an invocation of `<COMMAND>` for any `<COMMAND>` that does not start with a hyphen (including commands not specified in this document).
+None are required, but the runtime MAY support options that start with at least one hyphen.
+Global options MAY take positional arguments (e.g. `--log-level debug`).
+Command names MUST not start with hyphens.
```

Comment:
On Mon, Dec 07, 2015 at 10:13:31PM -0800, Mike Brown wrote:
> Yeah I just put it in the PR comment.  I refreshed the commit.  Not
> used to putting long winded commit messages in.

2b084a35 LGTM.  I'm fine with it landing as is, but here are a few
notes on the commit message:

  Discussed and worked on wording with @wking via private IRC.

This^ isn't important enough to include in the commit message.  Folks
reading this will care about what changed, what our motivation was,
and why we implemented it the way we did.  They don't care who we were
(except for copyright ownership, and Git's Author: covers that) or
what channel we used for the discussion.  I'll my Reviewed-by when
this lands, so folks will know that I signed-off on it (and we'll add
Reviewed-by tags for anyone else that LGTMs it too).

  "distinguish unrecognized commands…

This^ is missing a closing quote.

  `[1]` https…

No need to escape these^ references (e.g. see 3606bcf9).  Git's command
line tools don't markup the message, and GitHub links URLs but doesn't
render Markdown in the messages.

For motivating long-winded commit messages, see [1,2].  I like them
because you can go straight from ‘git blame’ to notes motivating that
line, instead of going blame → commit hash → GitHub PR → some comment
in that discussion.  And I have a local mirror of the commits (thanks
Git :), but I don't keep a local mirror of the GitHub PR discussion.

[1]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
[3]: http://chris.beams.io/posts/git-commit/#why-not-how